lib/sidekiq/cron/job.rb

```diff
@@ -294,7 +294,7 @@ def initialize input_args = {}
 
         # Get right arguments for job.
         @symbolize_args = args["symbolize_args"] == true || ("#{args["symbolize_args"]}" =~ (/^(true|t|yes|y|1)$/i)) == 0 || false
-        @args = args["args"].nil? ? [] : parse_args( args["args"] )
+        @args = parse_args(args["args"])
```

Comment:
:information_source: The check was unnecessary because `parse_args(nil)` is already `[]`